examples/create-env-cluster-topics/main.tf

```diff
@@ -90,7 +90,14 @@ module "honest_labs_kafka_topic_example_1" {
   service_account_id           = module.kafka_topic_service_account.service_account_id
   topic_name                   = "squad_raw_service_example_1_entity"
   connector_service_account_id = module.honest_labs_connector_service_account.service_account_id
-  depends_on                   = [module.cluster_admin_privilege_service_account]
+
+  max_compaction_lag_ms              = "9223372036854775807"
```

Comment:
Is this just a random number, or a real duration?  Let's use a real number as the example, with a comment showing the duration in more readable units.

Reply:
It is a real number. That's why I used in the test